### PR TITLE
NO-ISSUE: remove reports dir creation

### PR DESCRIPTION
**Pull request by osherdp**
Remove creation and usage of ``reports`` dir, as it's not really in use.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,8 +17,7 @@ autopep8:
 	autopep8 --recursive --in-place .
 
 pylint:
-	mkdir -p reports
-	PYLINTHOME=reports/ pylint release/
+	pylint release/
```
Comment (eliorerz) [inline Makefile:20]: Not related to this PR, But why using pylint and not flake8 as we use in test-infra?

Comment (osherdp) [reply Makefile:20]: We do both

**Review by eliorerz — APPROVED** [review]
LGTM

**Pull request merged**